package.json

```diff
@@ -47,8 +47,6 @@
     "date-fns": "^2.30.0",
     "emoji-regex": "^10.2.1",
     "emoji-toolkit": "^6.6.0",
-    "fnv-plus": "^1.3.1",
-    "tippy": "^0.0.0",
```

Comment:
Anyone know what this tippy version 0.0.0 is? very weird.

Reply:
@sanjeevdialpad https://github.com/dialpad/dialtone-vue/pull/1136